tcping.go

```diff
@@ -406,12 +456,24 @@ func tcping(tcpStats *stats) {
 			calculatedDowntime := calcTime(uint(math.Ceil(latestDowntimeDuration)))
 			color.Yellow.Printf("No response received for %s\n", calculatedDowntime)
 
+			/* Update startOfUptime */
+			tcpStats.startOfUptime = getSystemTime()
+
+			/* Calculate the longest downtime */
 			tcpStats.endOfDowntime = getSystemTime()
 			calcLongestDowntime(tcpStats)
 
 			tcpStats.wasDown = false
 		}
 
+		/* It means it is the first time to up */
```

Comment:
```suggestion
		/* It means it is the first time to go up */
```

Reply:
Thank you for your suggestion!
I have incorporated suggestions.